animation_nodes/libs/FastNoiseSIMD/source/compile_macos.sh

```diff
@@ -1,8 +1,17 @@
+#!/bin/sh
+set -e
+
 gcc -c FastNoiseSIMD.cpp -std=c++11 -fPIC -O3
 gcc -c FastNoiseSIMD_internal.cpp -std=c++11 -fPIC -O3
-gcc -c FastNoiseSIMD_sse2.cpp -std=c++11 -fPIC -O3 -msse2
-gcc -c FastNoiseSIMD_sse41.cpp -std=c++11 -fPIC -O3 -msse4.1
-gcc -c FastNoiseSIMD_avx2.cpp -std=c++11 -fPIC -O3 -march=core-avx2
+
+if [ "$(arch)" == "arm64" ]; then
+	gcc -c FastNoiseSIMD_neon.cpp -std=c++11 -fPIC -O3
+else
+	gcc -c FastNoiseSIMD_sse2.cpp -std=c++11 -fPIC -O3
+	gcc -c FastNoiseSIMD_sse41.cpp -std=c++11 -fPIC -O3
+	gcc -c FastNoiseSIMD_avx2.cpp -std=c++11 -fPIC -O3
```

Comment:
Why remove the compile flags `-msse2`, `-msse4.1`, and `-mavx2`? Are they not needed? And does Neon require similar flags?

Reply:
The MacOS x86 build is failing, so I think those flags are important.

Reply:
Sorry, that was some nonsense left from an earlier experiment... I hadn't tested the x86 build! Neon doesn't need any special flags here.